opsdroid/database/sqlite/__init__.py

```diff
@@ -0,0 +1,119 @@
+"""A module for sqlite database."""
+import os
+import logging
+import json
+import datetime
+import aiosqlite
+
+from opsdroid.const import DEFAULT_ROOT_PATH
+from opsdroid.database import Database
+
+
+class DatabaseSqlite(Database):
+    """A module for opsdroid to allow persist in a sqlite database."""
```

Comment:
We are in the process of replacing all the doc strings with the google style doc strings - reference #532 

I would like to have all the new code to be added to opsdroid core following this style, could you change the docstrings so they follow the google styling?